tools/windows-node-installer/pkg/cloudprovider/aws/ec2.go

```
@@ -273,9 +273,82 @@ func (a *awsProvider) findOrCreateSg(infraID string, vpc *ec2.Vpc) (string, erro
 	if err != nil {
 		return a.createWindowsWorkerSg(infraID, vpc)
 	}
+
+	log.V(0).Info(fmt.Sprintf("Using existing Security Group: %s.", sgID))
+
+	// Check winrm port open status for the exist security group
+	portOpenResult, err := a.checkSgWinrmPort(sgID)
+	if err != nil {
```

Why're we suppressing error here?

---

This is a typo. fixed.